options_test.go

```diff
@@ -132,7 +132,7 @@ func TestParseURL(t *testing.T) {
 		},
 		{
 			"postgres://vasya:pupkin@somewhere.at.amazonaws.com/postgres",
-			"somewhere.at.amazonaws.com:5432",
+			"somewhere.at.amazonaws.com:" + GetPGPort(),
```

Comment:
Given that these are not actually used for connections, and are purely for testing the parsing capability, i would not change any of these.

Reply:
Thanks. I think I went overboard with the changes, reverted.